packages/google_maps_flutter/pubspec.yaml

```diff
@@ -0,0 +1,30 @@
+name: google_maps_flutter_tizen
+description: Tizen platform implementation of google_maps_flutter
+homepage: https://github.com/flutter-tizen/plugins
+version: 0.1.0
+
+flutter:
+  plugin:
+    platforms:
+      tizen:
+        dartPluginClass: GoogleMapsPlugin
+        fileName: google_maps_flutter_tizen.dart
+
+dependencies:
+  flutter:
+    sdk: flutter
+  meta: ^1.3.0
+  google_maps_flutter_platform_interface: ^2.0.1
+  flutter_lints: ^1.0.4
+  stream_transform: ^2.0.0
+  webview_flutter: 2.0.4
```

Comment:
To pass the package validation, it should allow more than one version.
Could you review this and apply it?
```
dependencies:
    webview_flutter: ^2.0.4
```

Reply:
I have updated and checked it. Thank you.